### DISCUSSION TITLE
img in the github pages doens't appear

### DISCUSSION THREAD

**Discussion by DigKvs** [community/community#68487] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

i am coding a website of a e-comerce with react and tailwind and when i put the images of the project in the localhost its all normal but when i put the code in github pages all the images dissapear.

link to the site: https://digkvs.github.io/e-comerce_site

github folder: https://github.com/DigKvs/e-comerce_site

**Comment by melvincwng** [comment, marked as answer]
Hi @DigKvs, the reason why your images are not showing is because the `images` folder are not in the correct location.

**Try shifting your entire `images` folder inside the `public` folder.**

No other changes are required other than that. I've checked your image src path - they are of the correct relative file path as required by Vite.

Hope this resolves your issue and answers your question.

Take care, and have a good day ahead 😃 

Reference:
1. [Vite Documentation - the /public directory](https://vitejs.dev/guide/assets.html#the-public-directory)

**Comment by DigKvs** [comment]
Hi @melvincwng thanks for helping me, the solution works and the website is now perfectly.

Have a good day u 2! 😄

**Reply by melvincwng** [reply]
@DigKvs, always a pleasure to help 👍🏻
